src/features/members/components/invite-member-modal.tsx

```
@@ -267,22 +335,56 @@ export const InviteMemberModal = () => {
-	const isSeatsFull =
-		isPlanPaid && role !== "viewer" && usedSeats >= totalSeats;
+	const isSeatsFull = isPlanPaid && usedSeats >= totalSeats;
+	const newTotalSeats = totalSeats + seatsToAdd;
 
 	useEffect(() => {
 		if (!isSeatsFull) {
 			setSeatChangePending(false);
 		}
 	}, [isSeatsFull]);
 
+	useEffect(() => {
```

Biome (2.4.15)

[error] This hook specifies more dependencies than necessary: isSeatsFull.

(lint/correctness/useExhaustiveDependencies)